### ISSUE TITLE
MobArena integration fails.

### ISSUE THREAD

**Issue by mibby** [elBukkit/MagicPlugin#232]
Magic dev 2146
MobArena v0.98.1-SNAPSHOT (Compiled latest source as of commit https://github.com/garbagemule/MobArena/commit/6bbc9b55cff1ad8b9963c02209db545a6294fe10)

```
[Magic] MobArena integration failed, you may need to update the MobArena plugin to use Magic items
```

**Comment by NathanWolf** [comment]
Requires MobArena feature that hasn't been pulled yet, and may change. This is why I haven't released 6.9.9 yet.

**Comment by NathanWolf** [comment]
If you _really_ want to try it, you can grab the MobArena from my Jenkins server. But I think this will probably change, unfortunately garbagemule is hesitant to take my custom item PR, he wants to integrate into a larger custom reward system that seems like it's going to take a long while.

I'm going to look into other options, but I think anything else I do is going to be really hacky.
